xml/obs_build_config.xml

```diff
@@ -644,6 +644,20 @@ Support: pax debbuild</screen>
            </para>
         </listitem>
        </varlistentry>
+       <varlistentry>
+        <term><parameter>keepobsolete</parameter> (OBS 2.11 or later)</term>
+        <listitem>
+           <para>
+             The default behaviour of OBS is to remove binaries when a package object gets removed,
```

Comment:
instead of just "remove binaries", would it make sense to write "remove binaries from published repositories" (because OBS also offers a way to get at non-published binaries via "osc getbinaries")?

Even though I suppose "osc getbinaries" will not work after the package gets removed...

Reply:
we could .... however, it removes binaries everywhere, including published repositories. But also from build area, so it is not limited to published repos.... therefore I wrote this in this generic words.

Reply:
Ah, in that case, we should write "to remove all binaries".